Makefile

```diff
@@ -1,5 +1,5 @@
 
-ALL_DRIVERS := nifake nifgen nidcpower nidmm niswitch nimodinst
+ALL_DRIVERS := nifake niscope nifgen nidcpower nidmm niswitch nimodinst
```

Comment:
While the ordering here is not important, it appears random and also inconsistent throughout the codebase whenever we list the drivers.
How about we fix that?
(could be separate PR if you think it's cleaner)

Reply:
What is the preferred order? Alphabetical other than nifake first?

Reply:
I think order will be arbitrary no matter what. Let's just keep it consistent.
Maybe:

1. NI-FAKE
2. "Normal" drivers in alphabetical order.
3. NI-ModInst

?

Reply:
Changed